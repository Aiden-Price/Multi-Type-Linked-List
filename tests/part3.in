NEW 3
this
is
simple
NEW 4
this
is-nested
{0}
4.0 h
VIEW-NESTED 1
NEW 1
other
INSERT 0 2 {2}
REMOVE 0
DELETE 1 2
INSERT 0 2 {1}
INSERT 0 0 {1}
DELETE 0 0
VIEW-NESTED 0
