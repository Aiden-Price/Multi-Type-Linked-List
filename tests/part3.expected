List 0: this -> is -> simple
Nested 1: this -> is-nested -> {List 0} -> 4.0 h
this -> is-nested -> {this -> is -> simple} -> 4.0 h
List 2: other
INVALID COMMAND: INSERT
INVALID COMMAND: REMOVE
List 1: this -> is-nested -> 4.0 h
Nested 0: this -> is -> {List 1} -> simple
Nested 0: {List 1} -> this -> is -> {List 1} -> simple
Nested 0: this -> is -> {List 1} -> simple
this -> is -> {this -> is-nested -> 4.0 h} -> simple
